Source/relay/Guide/Items of the Month/Pizza Cube.ash

```diff
@@ -10,12 +10,17 @@ void IOTMPizzaCube(ChecklistEntry [int] resource_entries)
 
         // Entries
         string [int] description;
+        
+        boolean need_cheese = $item[goat cheese].available_amount() <3; // Obviously incomplete
 
-        if (fullness_limit() - my_fullness() > 3) {
-            description.listAppend(HTMLGenerateSpanOfClass("cheese/milk:", "r_bold") + " 3 goat cheese");
-            description.listAppend(HTMLGenerateSpanOfClass("lucky:", "r_bold") + " clover");
-            description.listAppend(HTMLGenerateSpanOfClass("warlike:", "r_bold") + " 3 of sonar-in-a-biscuit, Duskwalker syringe, cocktail napkin, unnamed cocktail, cigarette lighter, glark cable, short writ of habeas corpus");
+        if (need_cheese)
+        {
```

Comment:
Let's move curly braces to the same line. Eventually I'll write actual styling guides.
`if (need_cheese) {`